services/content_generator.py

```
@@ -418,94 +284,211 @@
+                if self._is_safe_url(clip_url):
+                    return f"**Clip:** [Watch]({clip_url})"
+                else:
+                    return f"**Clip:** Watch"
+        except Exception as e:
+            logger.warning(f"Failed to generate clip description: {e}")
+            if self._is_safe_url(clip_url):
+                return f"**Clip:** [Watch]({clip_url})"
+            else:
+                return f"**Clip:** Watch"
+    
+    def _generate_event_description(self, event_url: str, ai_service, force_ai: bool = False) -> str:
```

Ruff (0.12.2)

Unused method argument: `ai_service`

(ARG002)